### DISCUSSION TITLE
Grupo de SEO de Sites com JS: JavaScript Sites in Search Working Group

### DISCUSSION THREAD

**Discussion by fititnt** [frontendbr/forum#578] [Dicas]
[![image](https://cloud.githubusercontent.com/assets/812299/26238468/8e67e2a8-3c4f-11e7-923f-8e65641d3e94.png)](https://groups.google.com/forum/#!forum/js-sites-wg)

----

Google criou uma lista de e-mail específica para discutir impacto de sites que usam JavaScript nos mecanismos de busca em https://groups.google.com/forum/#!forum/js-sites-wg

**Isso é muito interessante para desenvolvedores avançados, e tende a ser um bom lugar para discutir assuntos pontuais sobre como tornar mais fácil para ser encontrado no Google.** Tem comentários muito pertinentes (para não dizer tristes) para quem usa JavaScript pesado para criar páginas que pretende indexar no Google.

**Comment by felipefialho** [comment]
Muito bom @fititnt
